webclient/app/src/app/features/trafficIncident/IncidentOverviewMap.jsx

```diff
@@ -73,22 +104,66 @@ function IncidentOverviewMap() {
             }
         }),
         // Add layer with Incident Icons
-        new IconLayer({
-            id: 'icon-layer',
-            data: trafficIncidents,
+        new ScatterplotLayer({
+            id: 'scatterplot-layer',
+            data: Object.entries(groupedIncidents),     // Using Object.entries to convert object to array of [key, value] pairs
+
+            pickable: true,     // Enable hover interactions with the icons
+            opacity: 0.8,       // 80% opacity for icons
+            stroked: true,      // Show border around icons
+            filled: true,       // Fill icons with color
+
+            // Size settings for markers
+            radiusScale: 15,
+            radiusMinPixels: 5,
+            radiusMaxPixels: 100,
+            lineWidthMinPixels: 1,
+
+            // Function to determine icon position
+            // d[1][0] contains the first incident in the group (all have same coordinates)
+            getPosition: d => [
+                d[1][0].cameraLongitude,
+                d[1][0].cameraLatitude
+            ],
+            getRadius: d => Math.sqrt(d[1].length) * 5,
+            getFillColor: d => getIconColor(d[1].length),
+            getLineColor: [0, 0, 0, 255],
+            onHover: info => {
+                if (info.object) {
```

Comment:
Why you need info.object?

Reply:
The `if (info.object) {...}` expression in the `onHover` event handler of the `ScatterplotLayer` component is used to check whether the user has actually moved the cursor over an object (marker) on the map.

When the user hovers the cursor over the `ScatterplotLayer`, the `onHover` callback function is called. This function gets an info object that contains information about the cursor hover event. 

If the cursor is not over the object (for example, it is over an empty spot on the map), info.object should be undefined or null.